models/intermediate/int__person.sql

```diff
@@ -19,9 +19,9 @@ SELECT
         WHEN upper(p.ethnicity) = 'NONHISPANIC' THEN 38003564
         ELSE 0
     END AS ethnicity_concept_id
-    , NULL AS location_id
-    , NULL AS provider_id
-    , NULL AS care_site_id
+    , cast(NULL AS INTEGER) AS location_id
```

Comment:
shall we use the macros for consistency?

Reply:
Good shout